### ISSUE TITLE
For liten kontrast på knapper i editeringsmodus på min profil

### ISSUE THREAD

**Issue by tormoseng** [AtB-AS/webshop#249]
**Beskrivelse av bug**
WCAG 1.4.3 Contrast

Ved å trykke på en av "Endre X"-knappene eller "Legg til t:kort" så kommer det opp en Avbryt og Lagre knapp. a11y-validator klager da på at fargekontrasten er for liten for WCAG 2 AA. Dette gjelder nok "hover-farge" på Avbryt vs bakgrunn og "hover-farge" på Lagre-knappen.

Eksempel for "Endre telefonnummer":
```
failureSummary: "Fix any of the following:↵  Element has insufficient color contrast of 3.4 (foreground color: #616161, background color: #000000, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
html: "<span class="typo-body__primary--bold ui-textContainer ">Avbryt</span>"

failureSummary: "Fix any of the following:↵  Element has insufficient color contrast of 1.92 (foreground color: #61aebc, background color: #007c92, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1"
html: "<span class="typo-body__primary--bold ui-textContainer ">Lagre</span>"
```

**Annen kontekst**
https://dequeuniversity.com/rules/axe/4.2/color-contrast?application=axeAPI

**Comment by gorandalum** [comment]
Jeg får ikke helt fargene dette blir evaluert til til å stemme. Spesielt den siste avviker stort fra hva som er mitt inntrykk.

Uten hover / Med hover
<div>
<img src="https://user-images.githubusercontent.com/675421/132015605-c5c0fdc3-4620-4990-a94f-8c773e941d3a.png">
<img src="https://user-images.githubusercontent.com/675421/132015602-b9f0d8f5-3c2c-46cd-a239-dd3c33b0c414.png"
</div>

Mens her er de fargene beskrevet i advarselen (#61aebc og #007c92) satt inn i en contrast checker:
![Screenshot 2021-09-03 at 15 48 51](https://user-images.githubusercontent.com/675421/132015932-3814d689-0667-4124-a518-ab9b3661d170.png)

Har du mulighet til å prøve med et annet sjekkeverktøy for en "second opinion" @tormoseng?

**Comment by tormoseng** [comment]
Enig i dine observasjoner. Ser ut til at feil farger blir plukket opp her.

Lukker saken!
